vj4/model/builtin.py

```diff
@@ -217,11 +212,30 @@
     PERM_EDIT_TRAINING_SELF
 )
 ADMIN_PERMISSIONS = PERM_ALL
+
+# Roles.
+ROLE_ROOT = 'root'
+ROLE_GUEST = 'guest'
+ROLE_DEFAULT = 'default'
+ROLE_MEMBER = 'member'
+ROLE_ADMIN = 'admin'
+
+BuiltinRoleDescriptor = functools.partial(
```

Comment:
1. It is a model (stipulates how data should looks like, instead of storing specific data)
2. It describes a role (our `role` is a type of `str` and this `BuiltinRoleDescriptor` describes attributes for the rule).

Reply:
Its not a role, why calling it "xxxRole"?

Reply:
`Descriptor` is ugly.

Reply:
descriptor is not ugly. There are descriptor everywhere, e.g. file descriptor (fd).